packages/common/src/api/README.md

```diff
@@ -0,0 +1,150 @@
+# audius-query
+
+## Making an api
+
+1. Call `createApi`
+
+   ```typescript
+   const userApi = createApi({
+     reducerPath: 'userApi',
+     endpoints: {
+       // ADD ENDPOINT DEFINITION HERE
+     }
+   })
+
+   export const {
+     /* NAMED HOOK EXPORTS */
+   } = userApi.hooks
+   export default userApi.reducer
```

Comment:
Wondering if we want to avoid default exports to adhere to the styleguide? Maybe can export as `reducer`